install_mhlib.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -eux
+
+ZIP_FILE=MultiHarp150_160_V3_1.zip
+MHLIB_DIR=MHLib\ v3.1.0.0
+DEMO_DIR=MultiHarp\ v3.1.0.0
+MHLIB_TAR_GZ=$MHLIB_DIR/Linux/MHLib_v3.1.0.0_64bit.tar.gz
+MHLIB=MHLib_v3.1.0.0_64bit
```

Comment:
we can't do that because the version number is not reliable here. for example, the zip file said `V3_1` but in the extracted folder named `v3.1.0.0`. so hard coded version is more maintainable for now.

Reply:
i guess it's too complicated for now. I don't trust the library developer can name it consistently. let's keep it.